test/common/network/address_impl_test.cc

```diff
@@ -25,24 +25,30 @@ void makeFdBlocking(int fd) {
   ASSERT_EQ(::fcntl(fd, F_SETFL, flags & (~O_NONBLOCK)), 0);
 }
 
-void testSocketBindAndConnect(const Instance& loopbackPort) {
+void testSocketBindAndConnect(const std::string& addrPortStr) {
+  auto addrPort = parseInternetAddressAndPort(addrPortStr);
```

Comment:
nit: We have been doing snake case for var names (addr_port). I think there might be a few other instances in this function.